content/articles/inbo_software/index.md

```diff
@@ -30,19 +30,19 @@ the package’s purpose and demonstrating its use.
 
 The following table gives a **quick overview**:
```

Comment:
I suggest to delete in the md file the many generated extra spaces (cf. my earlier comment: use the one from the Rmd file)